projects/ADuCM360_demo_cn0411/system.rteconfig

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<configuration xmlns:xs="http://www.w3.org/2001/XMLSchema-instance">
+<packages/>
+<device Dclock="16000000" Dcore="Cortex-M3" DcoreVersion="r2p0" Dendian="Little-endian" Dfamily="ADuCM36x Series" Dfpu="NO_FPU" Dmpu="NO_MPU" Dname="ADuCM360" Dvendor="Analog Devices:1" Pname="">
+<url>http://www.keil.com/dd2/analogdevices/aducm360</url>
+<package name="ADuCM36x_DFP" url="http://download.analog.com/tools/EZBoards/CM36x/Releases" vendor="AnalogDevices" version="1.0.2"/>
```

Comment:
It looks like these projects are using the older 1.0.2 CMSIS pack file. What are your thoughts on upgrading them to the current 1.0.3 CMSIS pack file?